From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Fri, 11 Jan 2013 11:41:00 +0200
Subject: [PATCH] packaging: reflect the corrected patch number handling

Fix spec files as GBP now treats 'Patch:' as patch number -1, instead of
0.

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>
---
 packaging/gbp-test2-alt.spec | 2 +-
 packaging/gbp-test2.spec     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/packaging/gbp-test2-alt.spec b/packaging/gbp-test2-alt.spec
index 155ea43..be5d8bc 100644
--- a/packaging/gbp-test2-alt.spec
+++ b/packaging/gbp-test2-alt.spec
@@ -8,7 +8,7 @@ License:    GPLv2
 Source10:   ftp://ftp.host.com/%{name}-%{version}.tar.gz
 Source:     foo.txt
 Source20:   bar.tar.gz
-# Gbp-Ignore-Patches: 0
+# Gbp-Ignore-Patches: -1
 Patch:      my.patch
 Patch10:    http://example.com/patches/my2.patch
 Patch20:    my3.patch
diff --git a/packaging/gbp-test2.spec b/packaging/gbp-test2.spec
index eec5d26..5b41aae 100644
--- a/packaging/gbp-test2.spec
+++ b/packaging/gbp-test2.spec
@@ -9,7 +9,7 @@ Source10:   ftp://ftp.host.com/%{name}-%{version}.tar.gz
 Source:     foo.txt
 Source20:   bar.tar.gz
 Source9999: gbp-test2-alt.spec
-# Gbp-Ignore-Patches: 0
+# Gbp-Ignore-Patches: -1
 Patch:      my.patch
 Patch10:    http://example.com/patches/my2.patch
 Patch20:    my3.patch
-- 
2.1.4

